packages/documentation/src/stories/foundation/elevation/elevation.components.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { ChangeEvent } from 'react';
+
+const backgrounds: { [key: string]: string } = {
+  'White': 'bg-white',
+  'Yellow': 'bg-yellow',
+  'Primary': 'bg-primary',
+  'Nightblue Bright': 'bg-nightblue-bright',
+  'Purple Bright': 'bg-purple-bright',
+};
+
+interface IState {
+  currentColor: string;
+}
+
+interface IBackgroundState {
+  backgroundHidden: boolean;
+}
+
+export class BackgroundSwitch extends React.Component<{}, IBackgroundState> {
+  constructor(props: {} | Readonly<{}>) {
+    super(props);
+    this.state = { backgroundHidden: false };
+  }
+
+  onChange(e: ChangeEvent<HTMLInputElement>) {
+    this.setState({ backgroundHidden: e.currentTarget.checked });
+    document
+      .querySelector<HTMLDivElement>('.elevation-container')
+      ?.style.setProperty('--post-bg-rgb', e.currentTarget.checked ? '255, 255, 255' : null);
+  }
+
+  render() {
+    return (
+      <fieldset className="mt-4">
+        <legend>Background visibility (for testing purposes)</legend>
+        <div className="form-check form-switch">
+          <input
+            className="form-check-input"
+            id="docs_Default_ExampleSwitch"
+            role="switch"
+            type="checkbox"
+            onChange={e => this.onChange(e)}
+          />
+          <label className="form-check-label order-first" htmlFor="docs_Default_ExampleSwitch">
+            Backgroud visible
+          </label>
+          <label className="form-check-label" htmlFor="docs_Default_ExampleSwitch">
+            Backgroud hidden
+          </label>
+        </div>
+      </fieldset>
+    );
+  }
+}
+
+export class ColorSwitchClass extends React.Component<{}, IState> {
+  constructor(props: {} | Readonly<{}>) {
+    super(props);
+    this.state = { currentColor: 'White' };
+  }
+
+  onChange(e: ChangeEvent<HTMLInputElement>) {
+    this.setState({ currentColor: e.currentTarget.value });
+    document
+      .querySelector('.elevation-container')
+      ?.setAttribute('class', `elevation-container ${backgrounds[e.currentTarget.value]}`);
```

Comment:
It does not work with dark mode.
Also, changing the background color sometimes produces white text on a white background, is this the desired behavior?